source/openapi3/payto-disputes.yml

```
@@ -1,7 +1,10 @@
 openapi: "3.0.2"
 info:
-  title: PayTo Disputes API
+  title: PayTo Disputes API (Not live)
   version: "1.0"
```

Shall we update the version too? 

```suggestion
  version: "0.0.1"
```

---

Good call, updated 🙌